content/zh-cn/contribute/_index.md

```diff
@@ -6,101 +6,116 @@ menu:
     weight: 10
 ---
 
-Cloud Native Glossary 内容存储在 [this GitHub repo](https://github.com/cncf/glossary) 中, 您可以在其中找到关于 glossary 的 [issues](https://github.com/cncf/glossary/issues), [PRs](https://github.com/cncf/glossary/pulls), 和 [discussions](https://github.com/cncf/glossary/discussions). 
+云原生词汇表的内容存储在这个 [GitHub 仓库](https://github.com/cncf/glossary)中，在这里你可以找到关于词汇表的 [议题](https://github.com/cncf/glossary/issues)、[PR](https://github.com/cncf/glossary/pulls) 和 [讨论](https://github.com/cncf/glossary/discussions) 的列表。
 
-您可以通过三种方式做出贡献:
+你可以通过四种方式做出贡献：
 
-1) 提出新术语
-2) 更新现有术语
-3) 帮助翻译词汇表
+1) [在现有的议题上工作](#在现有的议题上工作)
+2) [提出新术语](#提出新术语)
+3) [更新现有术语](#更新现有术语)
+4) [云原生词汇表翻译帮助](#云原生词汇表翻译帮助)
 
 ## 加入 Glossary 社区! 
-如果您想定期投稿, 请考虑加入我们的每月词汇表工作组会议. 您可以在 [CNCF calendar](https://www.cncf.io/calendar/) 找到会议详情. 您还可以在 CNCF Slack 上的 #glossary 频道中与维护者和其他贡献者联系 — 期待美好相遇! 
+如果你想定期做出贡献，可以考虑加入我们每月的词汇工作组会议。你可以在 [CNCF 日历](https://www.cncf.io/calendar/) 中找到会议细节。你也可以在 CNCF Slack 的 [#glossary](https://cloud-native.slack.com/archives/C02TX20MQBB) 频道中与维护者和其他贡献者联系，我们很乐意见到你! 
 
-## 提出新术语
-您可以提出一个新术语供其他人使用或自己创建一个新定义. 无论哪种方式, 您都将从创建问题开始(请注意, 您需要一个 GitHub 帐户来执行此操作).
+## 在现有的议题上工作
+进入云原生词汇表 GitHub 仓库的 [议题](https://github.com/cncf/glossary/issues)。在那里你会看到一个所有议题的列表。你可以通过标签来过滤（例如，English language, help needed, good first issue）。请注意，你需要一个 GitHub 账户来做这些事情。
+
+! [Issue 和 labels](/images/how-to/issue-and-labels.png)
+
+确保你感兴趣的术语还没有分配给某人。这里你可以看到，前三个术语是可用的，而下一个术语已经被分配了：
+
+![分配一个术语](/images/how-to/howto-04.png)
 
-以下是针对尚未熟悉 GitHub 的人的分步指南. **如果你是 GitHub Pro 账户**, 请快速查看以确保您使用我们的问题模板、适当的命名约定、在 Slack 上声明 PR（否则我们可能会错过它）以及在哪里可以找到文件模板. 在开始之前, 请务必阅读 [Style Guide](/zh-cn/style-guide/).
+一旦你找到了一个你想做的术语，就在议题中说出来。点击它并添加一个评论。
 
-### 创建 issue
-前往 [Glossary GitHub repo](https://github.com/cncf/glossary/issues) 并点击 "new issue".
+![提出一个议题](/images/how-to/claiming-an-issue.png)
+
+此外，请加入 CNCF Slack 的 [#glossary](https://cloud-native.slack.com/archives/C02TX20MQBB) 频道，让维护者知道你为一个新术语提出了一个议题（最好标记 <em>@Catherine Paganini</em>, <em>@jmo</em>, <em>@Seokho Son</em>, <em>@Jihoon Seo</em>, 和/或 <em>@iamnoah<em> 以确保他们不会错过它）。请注意，你一次只能认领一个术语。如果你想做多个术语，请在申请下一个术语之前完成前一个术语。
+
+一旦他们把它分配给你，你就可以开始工作了。接下来的步骤，请参考 [提交新术语（创建PR）](#提交新术语（创建PR）)部分。
+
+## 提出新术语
+你可以提出一个新的术语让别人去研究，或者自己创造一个新的定义。无论哪种方式，你都要从创建一个议题开始。请注意，术语必须符合 [CNCF的云原生定义](https://github.com/cncf/toc/blob/main/DEFINITION.md)。唯一的例外是理解云原生概念所需的基础性术语。
 
-![issues](/images/how-to/howto-01.png)
+以下是为那些还不熟悉 GitHub 的人提供的分步指南。**如果你是 GitHub 的专业人员**，请**特别**看一下，确保你使用我们的议题模板，适当的命名惯例，在 Slack 上声明一个 PR（否则我们可能会错过它），以及在哪里找到文件模板。在开始之前，请确保阅读 [风格指南](/style-guide/)--谢谢! 
```

Comment:
Since we now have the Chinese version of "Style Guide",
please update all the links under `content/zh-cn/` to point the Chinese version!